src/main/java/org/mybatis/caches/redis/SerializeUtil.java

```diff
@@ -29,30 +25,36 @@ private SerializeUtil() {
     }
 
     public static byte[] serialize(Object object) {
-        ObjectOutputStream oos = null;
-        ByteArrayOutputStream baos = null;
-        try {
-            baos = new ByteArrayOutputStream();
-            oos = new ObjectOutputStream(baos);
-            oos.writeObject(object);
-            return baos.toByteArray();
-        } catch (Exception e) {
-            throw new CacheException(e);
-        }
+    	 try {
```

Comment:
@laddcn Is there a better way to handle this?  Try/catch logic is expensive way to have two types of serializators.

Reply:
Thank you for review. I have modified this problem and committed as following. 
1. For the first time a new class occurs, register kryo default serializer first;  if default kryo serializer fails, register  javaSerializer or externalizableSerializer as a fallback
2. For class ever occurred, serialize directly
3. Use a hashset to store classes occurred
4. Unit tests work fine
The above changes are base on the chapter "Using standard Java Serialization" in the home page of kyro
https://github.com/EsotericSoftware/kryo